Docker/README.md

```diff
@@ -66,13 +66,16 @@ The [tags](https://hub.docker.com/r/freshrss/freshrss/tags) correspond to FreshR
 
 * `:latest` (default) is the [latest stable release](https://github.com/FreshRSS/FreshRSS/releases/latest)
 * `:edge` is the rolling release, same than our [git `edge` branch](https://github.com/FreshRSS/FreshRSS/tree/edge)
-* `:x.y.z` are [specific FreshRSS releases](https://github.com/FreshRSS/FreshRSS/releases)
-* `:arm` or `:*-arm` are the ARM `arm32v7` versions (e.g., for Raspberry Pi).
+* `:x.y.z` tags correspond to [specific FreshRSS releases](https://github.com/FreshRSS/FreshRSS/releases), allowing you to target a precise version for deployment
+* `:x.y` tags are tied to a specific major version and minor version number. For example, `:12.2` will automatically receive updates for any `12.2.x` releases, but will not update to `12.3.x`
+* `:x` tags track the latest release within a major version series. For instance, `:12` will update to include any `12.x` releases, but will exclude versions beyond `13.x`
+* `*-alpine` use apline as base-image instead debian
+* Our Docker images are designed with multi-architecture support, accommodating a variety of Linux platforms including `linux/arm/v7`, `linux/arm64`, and `linux/amd64`.
   * For other platforms, see the [custom build section](#build-custom-docker-image)
 
 ### Linux: Debian vs. Alpine
 
-Our default image is based on [Debian](https://www.debian.org/). We offer an alternative based on [Alpine](https://alpinelinux.org/) (with the `:alpine` or `*-alpine` tag suffix).
```

Comment:
Don't we have the `freshrss/freshrss:alpine` tag anymore?

Reply:
when I am correct we have `latest-alpine`or `v2-alpine` and so on

Reply:
![grafik](https://github.com/FreshRSS/FreshRSS/assets/1573025/da58321d-67b6-40f6-9ef4-1195830eb53e)
I testet it... there is something odd

Reply:
![grafik](https://github.com/FreshRSS/FreshRSS/assets/1573025/20d2ed94-cb6c-4fdb-b3f9-96ba6792c004)
working now

Reply:
So `freshrss:latest-alpine`, but no `freshrss:alpine`?

Reply:
yes

Reply:
Can you think of a way to also have `freshrss:alpine` ?
It seems to be the more common way, e.g. https://hub.docker.com/layers/library/httpd/alpine/images/sha256-e4e78032a4c6375e4e697d68f9e70306bce66ccdf09659d4c06b029d5fc60d41?context=explore

Reply:
done, but I find `latest-alpine` whould be "nice"

Reply:
![grafik](https://github.com/FreshRSS/FreshRSS/assets/1573025/c42054ca-6746-4555-b6d8-49b745133695)